src/mpcCoreKit.ts

```diff
@@ -245,7 +247,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
         locationReplaceOnRedirect: true,
         serverTimeOffset: this.options.serverTimeOffset,
         keyType: this.keyType,
-        useDkg: this.options.useDkg,
+        useDkg: this.keyType === KeyType.ed25519 && this.options.useDkg === undefined ? true : this.options.useDkg,
```

Comment:
```suggestion
        useDKG: this.keyType === KeyType.ed25519 && this.options.useDkg === undefined ? true : this.options.useDkg,
```

normally, `DKG` is all upper case